src/components/OSCALProfileCatalogInheritance.js

```diff
@@ -0,0 +1,81 @@
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import { TreeView, TreeItem } from "@material-ui/lab";
+import ExpandMoreIcon from "@material-ui/icons/ExpandMore";
+import ExpandLessIcon from "@material-ui/icons/ExpandLess";
+import { Grid, IconButton, List, Paper } from "@material-ui/core";
+import ListSubheader from "@material-ui/core/ListSubheader";
+
+const useStyles = makeStyles((theme) => ({
+  inheritance: {
+    marginTop: theme.spacing(2),
+    marginBottom: theme.spacing(2),
+  },
+}));
+
+function renderTree(nodes, id) {
+  if (nodes.inherited == null) {
+    return null;
+  }
+
+  const generateLabel = (title, type) =>
+    type === "profile" ? `Profile: ${title}` : `Catalog: ${title}`;
+
+  const idReassign = id;
+  const children = [];
+
+  nodes.inherited.forEach((node) => {
+    idReassign[0] += 1;
+    children.push(
+      <TreeItem
+        nodeId={idReassign[0].toString()}
+        label={generateLabel(node.title, node.type)}
+        expandIcon={
```

Comment:
I chose not to because if we use the `TreeView` default icon properties, then all icons will have the same label. That won't go well for testing purposes so if we want to be able to test the expansion and collapse of the tree, then we can't use the `TreeView` defaults. If we don't care about that, then the defaults are fine.

Reply:
Ideally we don't want to be required to rely on aria labels, instead taking an approach of finding the expected text then the closest button.

As mentioned elsewhere on this PR, we also want the tree to be expanded by default, so I'm not sure will need to test clicking on these at all.